conda_forge_tick/auto_tick.py

```diff
@@ -502,7 +502,7 @@ def migrator_status(
     import graphviz
     from streamz.graph import _clean_text
 
-    gv = graphviz.Digraph(graph_attr={"packmode": "node"})
+    gv = graphviz.Digraph(graph_attr={"packmode": "array_3"})
```

Comment:
Commit message was supposed to be `Use graphviz packmode=array_3`, of course. (Lazy copy-pasting..)